cardlibrary/baselibrary.lua

```diff
@@ -1960,7 +1960,7 @@ local base = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Green"] = 1,["Neutral"] = 1,},
 		["Effect"] = {
 			Name = "Render",
-			Description = "Whenever BadBreadMaker is targeted by a creature, that creature and BadBreadMaker gain 300 health.",
+			Description = "Whenever BadBreadMaker is targeted by a fighter, increase the health of it and BadBreadMaker by 300.",
```

Comment:
increase the health of that fighter and badbreadmaker by 300.
We need to get rid of ambiguity.